### ISSUE TITLE
Libadwaita(Apps such as Nautilus, gnome-extensions) 

### ISSUE THREAD

**Issue by WitheRosE901** [vinceliuice/Graphite-gtk-theme#88]
I see that you temporarily solved this issue with the `-i` option, but I am going to recommend that the `install.sh` file do this by default. As far as I know, this is the only way to apply the theme to libadwaita apps such as Nautilus, Settings, and gnome-extensions.

So, I found the theme in the gnome-look website, Graphite is top rated there, and I installed it using ocs-url. Had an issue where libadwaita apps would not get the theme applied. So I consulted this repository and found out about it. Now, I do not know how you would get it to work with ocs-url installation, so that gnome-look website installation works perfectly, so I'll ask here.

**Comment by vinceliuice** [comment]
Nope, ocs-url installation can not apply the theme to libadwaita apps, because `-l` option is not a perfect way to theme libadwaita apps, I just link the gtk-4.0 theme to user's gtk-4.0 config folder then it can force all gtk-4.0 app to use this theme, but ocs-url installation will not do this
